src/javascript/app_2/App/Components/Elements/PositionsDrawer/ProgressSlider/positions-progress-slider.jsx

```diff
@@ -0,0 +1,58 @@
+import classNames            from 'classnames';
+import PropTypes             from 'prop-types';
+import React                 from 'react';
+import ProgressTicks         from './positions-progress-ticks.jsx';
+import RemainingTime         from '../../../../Containers/remaining-time.jsx';
+
+const ProgressSlider = ({
+    className,
+    has_result,
+    ticks_count,
+    current_tick,
+    percentage,
+    remaining_time,
+}) => {
+    if (!percentage && !ticks_count || has_result) return <div className='progress-slider--completed' />;
+    return (
+        <div className={classNames('progress-slider', className)}>
+            {(ticks_count < 5) ?
```

Comment:
Why `5`?

Reply:
Because design isn't ready yet for ticks duration and after testing it only makes sense to show duration slider in place ticks slider for `tick_counts` above 5.  Regardless this is still a work in progress and will be handled in the subsequent card/pr.